manage_sections/migrations/0001_initial.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+LTI_PERMISSIONS_DATA = [
+    ('manage_sections', '*', 'Account Observer', True),
```

Comment:
Some strange formatting issues here -- probably aren't an issue, but Python does care about indentation, so might be worth fixing.